### PR TITLE
SPCTR-803 : Fix: price list css

### PR DESCRIPTION
**Pull request by jamyrbsf**
### Description
All issues mention for price list in Spectra Acceptance Testing R:3

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by bsf-zanev — APPROVED** [review]
All Updated Code Reviewed and Approved

File: src/blocks/price-list/style.scss [hunk 1]
```diff
@@ -100,10 +107,18 @@
...
 
-		&.uagb-rm__align-right .uagb-rm-details {
-			display: flex;
-			flex-direction: column;
-			width: 100%;
+		&.uagb-rm__align-right {
+
+			// & .uagb-rm-details {
+			// 	display: flex;
+			// 	flex-direction: column;
```
Comment (RajkiranBagal) [inline src/blocks/price-list/style.scss:114]: Ig this code is not needed we can remove it. Confirm it once.

File: src/blocks/price-list/editor.lazy.scss [hunk 2]
```diff
@@ -32,6 +32,11 @@
 			box-sizing: border-box;
 		}
 
+		& .uagb-rest_menu__wrap .uagb-rm__content {
+			padding-left: unset;
```
Comment (RajkiranBagal) [inline src/blocks/price-list/editor.lazy.scss:36]: Why do we need this CSS? Please confirm it once.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/price-list/style.scss (hunk 1 above)
Comment (jamyrbsf) [reply src/blocks/price-list/style.scss]: done

File: src/blocks/price-list/editor.lazy.scss (hunk 2 above)
Comment (jamyrbsf) [reply src/blocks/price-list/editor.lazy.scss:36]: as this code is not organized i have added this as a extra step

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**